src/main/scala/rocket/HellaCache.scala

```diff
@@ -51,8 +40,8 @@ trait HasL1HellaCacheParameters extends HasCacheParameters with HasCoreParameter
 
   require(isPow2(nSets))
   require(rowBits >= coreDataBits)
-  require(rowBits <= outerDataBits)
-  require(xLen <= outerDataBits) // would need offset addr for puts if data width < xlen
+  require(rowBits == cacheDataBits) // TODO should rowBits even be seperably specifiable?
```

Comment:
I agree that TL width and coreInstBits are both lower bounds on rowBits.  But we might want it to be wider, to amortize the other costs in the I$ readout, like the tag check, over multiple accesses.  Since that decision is exposed to the RTL, it can't be handled in the backend.

If it's really a pain, we can ditch it, though.